### ISSUE TITLE
Revise black focus outlines in welcome experience

### ISSUE THREAD

**Issue by rossmoody** [brave/brave-browser#4658]
Need to revise the focus styling in the welcome experience away from the thick black squares. 

![image](https://user-images.githubusercontent.com/29072694/58682265-add52880-8324-11e9-957d-c918b9ee5c31.png)

**Comment by bsclifton** [comment]
Related / dupe of https://github.com/brave/brave-browser/issues/3633

**Comment by kjozwiak** [comment]
@rossmoody can you please add a screenshot/example of how the new focus lines should appear under `brave://welcome`. This way QA will be able to tell if what we're seeing is actually the expected/intended behaviour 👍 

Were the focus lines changes anywhere else, or this strictly for the `brave://welcome` page?

**Comment by btlechowski** [comment]
Verification passed on

Brave | 0.68.128 Chromium: 76.0.3809.100 (Official Build) beta (64-bit)
-- | --
Revision | ed9d447d30203dc5069e540f05079e493fc1c132-refs/branch-heads/3809@{#990}
OS | Ubuntu 18.04 LTS

The ugly focus outline is gone. Checked all welcome sub-pages.

![image](https://user-images.githubusercontent.com/34715963/62984526-d94fb580-be33-11e9-96f9-3d48561a5ca2.png)
